### PR TITLE
myEcommerce_homepage

### PR DESCRIPTION
**Pull request by saintjohnie**
html task Homepage
I created an homepage of my ecommerce website with just html and css

### CODE REVIEW

File: submissions/NwagboJohn/homepageTask/index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
+<head>
+    <title>Johnie_ecom</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="head">
+        <img src="https://i.ibb.co/grnrJXk/conflicting-copy.png" alt="logo" id='logo' class=logo>
```
Comment (adaugochi) [inline submissions/NwagboJohn/homepageTask/index.html:8]: What if I don't have an internet connection, it means I can't view this image on my browser.  Next time, I will advise you to download this image. Then create an `image` folder within your `homePageTask` folder and place the image you need for this project there and point to that path on your `src` attribute. Same goes to other parts of code where `links` are used as the image src attribute

File: submissions/NwagboJohn/javascriptTask/index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <title>Johniestore</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <h1>Products</h1>
+        <div class="displayItem">
+        </div>
+        <button id="button">next</button>
+        <template id="product">
```
Comment (kenny-wealth) [inline submissions/NwagboJohn/javascriptTask/index.html:14]: Nice use of template 👍

File: submissions/NwagboJohn/javascriptTask/myJavascript.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        img.src = product.imageUrl;
+
+        var name = prod.querySelector(".prodName");
+        name.textContent = product.name;
+
+        var price = prod.querySelector(".prodPrice");
+        price.textContent = "₦" + product.price.toLocaleString("en-NG");
+
+        displayItem.appendChild(prod);
+    });
```
Comment (kenny-wealth) [inline submissions/NwagboJohn/javascriptTask/myJavascript.js:38]: You may disable the button after the first load of products to stop a repetitive loading of products

File: submissions/NwagboJohn/javascriptTask/style.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
+ @import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+            @import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
```
Comment (kenny-wealth) [inline submissions/NwagboJohn/javascriptTask/style.css]: These don't need to be indented as much

**Review by kenny-wealth — COMMENTED** [review]
See comments

File: submissions/NwagboJohn/javascriptTask/myJavascript.js (hunk 3 above)
Comment (saintjohnie) [reply submissions/NwagboJohn/javascriptTask/myJavascript.js:38]: done

File: submissions/NwagboJohn/javascriptTask/style.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
+ @import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
+ @import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
+
+            body {
```
Comment (kenny-wealth) [inline submissions/NwagboJohn/javascriptTask/style.css]: These CSS rules should not be indented. Please revise

**Review by kenny-wealth — CHANGES_REQUESTED** [review]

**Review by kenny-wealth — APPROVED** [review]

**Review by adaugochi — APPROVED** [review]

File: submissions/NwagboJohn/homepageTask/index.html (hunk 1 above)
Comment (saintjohnie) [reply submissions/NwagboJohn/homepageTask/index.html:8]: Noted and thank you

**Pull request merged**